.ruff.toml

```diff
@@ -27,6 +27,7 @@ ignore = [
   "PLR2004", # Magic value used in comparison
   "TRY003", # Avoid specifying long messages outside the exception class
   "UP038",  # Use X | Y in isinstance check instead of (X, Y)
+  "PLR0911", # too many return statements
```

Comment:
Can we inline this ignore to the specific statement where we want to ignore this error type?

Reply:
we can, but I think we can ignore it everywhere in the same way we are ignoring "too many branches"